composer.json

```diff
@@ -13,7 +13,9 @@
         "sparkphp/command": "self.version"
     },
     "require": {
-        "php": ">=5.5"
+        "php": ">=5.5",
+        "equip/assist": "^0.1.2",
+        "equip/data": "^2.3"
```

Comment:
What it could look like isn't difficult to visualize, but that doesn't address the points I expressed above. How many use cases are we likely to encounter where we'd want to use the immutable traits apart from the repository and entity implementations, even assuming this repository did serve as a solitary example? I'm not convinced it's worth the trouble of separation and maintenance versus pulling in a bit more code from api-commands than might be needed for this particular use case.